cwms-data-api/src/test/java/cwms/cda/api/location/kind/OutletControllerTestIT.java

```diff
@@ -96,6 +114,7 @@ public static void tearDown() throws Exception {
 
             outletDao.deleteOutlet(EXISTING_CONDUIT_GATE.getOfficeId(), EXISTING_CONDUIT_GATE.getName(),
                                    DeleteRule.DELETE_ALL);
+            deleteLocation(context, RATED_OUTLET_LOCATION.getOfficeId(), RATED_OUTLET_LOCATION.getName());
```

Comment:
Add location group deletion just in case.  Use the ID in the NEW_RATED_OUTLET object.

Reply:
This still needs to be resolved

Reply:
Added location group removal